cmd/simRecon/simRecon.go

```diff
@@ -31,7 +34,10 @@ func SimulateEvolve(rootFastaFile string, treeFile string, gp string, simOutFile
 
 //ReconstructSeq takes in a newick tree and leaf sequences and returns a reconstructed tree
 func ReconstructSeq(newickInput string, fastaInput string, outputFilename string) {
-	tree := expandedTree.ReadTree(newickInput, fastaInput)
+	tree, err := expandedTree.ReadTree(newickInput, fastaInput)
+	if err != nil {
```

Comment:
same error check comment as above.  I will not leave any more comments, but I see this at least one more place in the code.